### PR TITLE
Enable svdkl example

### PR DESCRIPTION
**Pull request by fehiepsi**
The fix is already available in pytorch nightly, so it is fine to enable the example now.

### CODE REVIEW

**Comment by neerajprad** [comment]
We are back to the same issue as earlier - maybe there is another source of slowdown in the example?

**Comment by fehiepsi** [comment]
I'll merge dev and watch travis this time. I have no idea for this bug. :(

**Comment by fehiepsi** [comment]
This is closed because the slowness is still flaky in Travis.

File: examples/contrib/gp/sv-dkl.py [hunk 1]
```diff
@@ -101,8 +101,8 @@ def cnn_fn(x):
     # CNN will transform a high dimension image into a low dimension 2D tensors for RBF kernel.
     # This kernel accepts inputs are inputs of CNN and gives outputs are covariance matrix of RBF
     # on outputs of CNN.
-    kernel = gp.kernels.Warp(gp.kernels.RBF(input_dim=10, lengthscale=torch.ones(10)),
-                             iwarping_fn=cnn_fn)
+    kernel = gp.kernels.Warping(gp.kernels.RBF(input_dim=10, lengthscale=torch.ones(10)),
```
Comment (neerajprad) [inline examples/contrib/gp/sv-dkl.py]: We should still get this fix in!

File: tests/test_examples.py [hunk 2]
```diff
@@ -21,9 +21,7 @@
     'contrib/autoname/scoping_mixture.py --num-epochs=1',
     'contrib/autoname/mixture.py --num-epochs=1',
     'contrib/autoname/tree_data.py --num-epochs=1',
-    skipif_param('contrib/gp/sv-dkl.py --epochs=1 --num-inducing=4',
-                 condition='CI' in os.environ,
-                 reason='https://github.com/uber/pyro/issues/1540'),
+    'contrib/gp/sv-dkl.py --epochs=1 --num-inducing=4',
```
[the remark this answers is not in the document]
Comment (fehiepsi) [reply tests/test_examples.py]: I test with python 2.7:
+ torch 0.4: without pyro gp: 42s; with pyro gp: 79s
+ torch 1.0.20181201: without pyro gp 79s; with pyro gp 114s

Note that my machine is very unstable so the above stats is not so reliable.

**Pull request closed without merging**